consul/consul.go

```diff
@@ -0,0 +1,35 @@
+package consul
+
+import (
+	"fmt"
+	"github.com/hashicorp/consul/api"
+	"log"
+)
+
+// GetNodes return a list of node addresses from a given Consul service
+func GetNodes(service string, consulAddr string, dc string) []string {
+	defaultConfig := api.DefaultConfig()
+	// We use default config because it can be fine configured through ENV variables
+	// (such as auth), in most case we only need to change the address
+	defaultConfig.Address = consulAddr
+	client, err := api.NewClient(defaultConfig)
+	if err != nil {
+		panic(err)
```

Comment:
Can you return an err instead of a panic, depending of where we call the GetNodes in the future is could be domageable.